lib/config/config-file.js

```diff
@@ -115,6 +115,11 @@ function loadJSONConfigFile(filePath) {
     } catch (e) {
         debug(`Error reading JSON file: ${filePath}`);
         e.message = `Cannot read config file: ${filePath}\nError: ${e.message}`;
+        e.messageTemplate = "failed-to-parse-json";
```

Comment:
I don't think the error here will necessarily be a JSON parsing error. This error-handling code will run when anything in `JSON.parse(stripComments(readFile(filePath)))` throws an error, which could include errors from the filesystem.

This could either be addressed by using different `try`/`catch` blocks for the different function calls (and providing a different error message as appropriate), or making the `failed-to-parse-json` error message more generic (e.g. "Failed to read JSON file" rather than "Failed to parse JSON file").